Classes/Settings/DefaultReactionDetailController.swift

```diff
@@ -18,6 +22,8 @@ class DefaultReactionDetailController: UITableViewController {
     @IBOutlet var heartCell: UITableViewCell!
     @IBOutlet var enabledSwitch: UISwitch!
 
+    var listener: DefaultReactionListener?
```

Comment:
`weak` to avoid retain cycle! Also the protocol + var should just use the term "Delegate" b/c this is a 1:1 (not 1:n)

Reply:
Done! Thx for pointing out.